app/controllers/spree/admin/gift_cards_controller.rb

```diff
@@ -1,17 +1,19 @@
 class Spree::Admin::GiftCardsController < Spree::Admin::BaseController
-  before_filter :load_gift_card_history, only: [:show]
   before_filter :load_user, only: [:lookup, :redeem]
   before_filter :load_gift_card_for_redemption, only: [:redeem]
   before_filter :load_gift_card_by_id, only: [:edit, :update, :send_email]
   before_filter :load_order, only: [:edit, :update]
+  Spree::LineItem.whitelisted_ransackable_associations += %w[order]
```

Comment:
@allisonlarson usually we do those at the model level, since it is a global change